forge-ai/src/main/java/forge/ai/AiAttackController.java

```diff
@@ -850,6 +850,23 @@ public final int declareAttackers(final Combat combat) {
         // Determine who will be attacked
         GameEntity defender = chooseDefender(combat, bAssault);
 
+        // Avoid creating an enemy army
+        // Darien, King of Kjeldor; Kazuul, Tyrant of the Cliffs etc
+        if (!bAssault && defender instanceof Player) {
+            for (Card c : ((Player)defender).getCardsIn(ZoneType.Battlefield)) {
+                for (Trigger t : c.getTriggers()) {
+                    if ((TriggerType.Attacks.equals(t.getMode())
```

Comment:
this seems too broad and also overlaps partly with more precise checks in `doRevengeOfRavensAttackLogic` and `isEffectiveAttacker` anyway

while it's usually preferable to try and parse it by code alone I wonder if a name-based check will be cleaner here
- or if there are some more cards maybe `AiCombatAggressionMod` SVar approach would even be beneficial